### DISCUSSION TITLE
VEIKK A50 not detected

### DISCUSSION THREAD

**Discussion by muhammadtaufik92** [OpenTabletDriver/OpenTabletDriver#1554] [Q&A] [answered]
As we know OpenTabletDriver supports VEIKK A50 but the tablet is not detected. I've tried to uninstall the VEIKK driver and the result is still the same. Maybe I need such a USB cleanup or anything thad I dont know. Thanks for your will to help :)

**Comment by jamesbt365** [comment, marked as answer]
(More an issue thing) Export a diagnosis, I don't know what is going on til then. Help > export diagnosis, save it as a txt and send it here if you can, if not make an issue or send it to our discord.

Assuming this is resolved, make sure otd is up to date, make sure your cable and port is fine and you may need to clean veikk drivers manually in device manager.
